platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/task/WorkflowTaskNotificationSubmitter.java

```diff
@@ -4,23 +4,32 @@
 import com.softicar.platform.core.module.email.EmailContentType;
 import com.softicar.platform.core.module.email.buffer.BufferedEmailFactory;
 import com.softicar.platform.core.module.page.PageUrlBuilder;
+import com.softicar.platform.core.module.user.AGUser;
 import com.softicar.platform.workflow.module.WorkflowI18n;
 
 public class WorkflowTaskNotificationSubmitter {
 
 	private final AGWorkflowTask task;
+	private final AGUser user;
 
 	public WorkflowTaskNotificationSubmitter(AGWorkflowTask task) {
 
 		this.task = task;
+		this.user = task.getUser();
+	}
+
+	public WorkflowTaskNotificationSubmitter(AGWorkflowTask task, AGUser user) {
```

Comment:
I think we should take a slightly different approach here, and replace this new constructor with a method:
```
public WorkflowTaskNotificationSubmitter setNotificationRecipient(AGUser notificationRecipient) {
    this.notificationRecipient = Objects.requireNonNull(notificationRecipient);
    return this;
}
```
The above constructor would still assign `this.notificationRecipient = task.getUser();`, ofc.